### ISSUE TITLE
feat: single file export containing all the Pokemon moved to the virtual HOME boxes

### ISSUE THREAD

**Issue by RodrigoTR04** [andrewbenington/OpenHome#34]
Description
The idea is to be able to dump all the Pokemon from multiple save files into the virtual HOME boxes and then afterwards be able to save it all into a single file (instead of having to extract each individual Pokemon into their own file and keep it inside a folder). Then later be able to reopen it for viewing and editing.

**Comment by evanwporter** [comment]
Yeah this is a good idea though I think it would be better if you could dump Pokemon from a SINGLE save as opposed to multiple since who has more than 2-3 saves worth of pokemon? But until this is implemented the best solution is to dump the pokemon into the pkdb via PKHex. Then add them en masse by clicking and dragging.

**Comment by RodrigoTR04** [comment]
Dumping from a single file is easy enough, you dump it all from the save file to individual files with PKHeX and then import to OpenHome, having a system that inports from multiple files could be very convenient. Also 2-3 save files worth of pokemon? A lot of people have much more than that. I've completed the Pokedex in each of the games, so there's a lot there already. Then I kept all the save files from my childhood (spanning from generation 3 and onwards), plus a lot of later playthroughs that I did just because I wanted to have fun. Overall I personally have over 50 save files saved, probably totaling a couple dozen thousand pokemon in boxes.

**Comment by evanwporter** [comment]
@RodrigoTR04 this feels very doable since all of the Save read/write functionality has been implemented. For now this could be implemented as a script until we figure out where to fit it into OpenHome. A lot of the OpenHome backend is being refactored as we move from a pure TS => Rust backend, so this would likely need to wait until #63 is done. As for you other request in #36, this could be done by via a parameter to the script. As for you lookup feature this will take even longer as its much more involved (but still doable).
